test/unit/initialize.spec.ts

```diff
@@ -555,7 +609,10 @@ describe("Initialization", () => {
             const endIndex = flicking.getIndex();
             const indexAtDestPos = Math.floor(destPos / panelWidth);
             // As all delta is above threshold, it should change panel at least once
-            const expectedIndex = Math.min(nearestPanel.getIndex() + snapCount, Math.max(1, indexAtDestPos));
+            const expectedIndex = destPos >= 100
+              ? Math.min(nearestPanel.getIndex() + snapCount, Math.max(1, indexAtDestPos))
+              // Sometimes, simulation won't return proper dest position for given delta.
```

Comment:
I suggest to prepend `TODO:` to check later. I think this code may have potential problem.

Reply:
It's been changed at other PR(#203). That's quite more detailed, so please check it after I rebase it.